concepticondata/conceptlists/Luangthongkum-2019-341.tsv

```diff
@@ -0,0 +1,342 @@
+Luangthongkum-2019-341-23	23	bite (v.)	1403	BITE
+Luangthongkum-2019-341-24	24	bitter	887	BITTER
+Luangthongkum-2019-341-25	25	blood	946	BLOOD
+Luangthongkum-2019-341-26	26	blow (a fire)	176	BLOW (WITH MOUTH)
+Luangthongkum-2019-341-27	27	boar (wild~)	3154	WILD BOAR
+Luangthongkum-2019-341-28	28	boat	1844	BOAT
+Luangthongkum-2019-341-29	29	body dirt		
+Luangthongkum-2019-341-30	30	boil (v.)	2493	BOIL
+Luangthongkum-2019-341-31	31	bone	1394	BONE
+Luangthongkum-2019-341-32	32	braid		
+Luangthongkum-2019-341-33	33	brain	1583	BRAIN
+Luangthongkum-2019-341-34	34	bran,chaff	2363	CHAFF (HUSK)
+Luangthongkum-2019-341-35	35	breasts	1402	BREAST
+Luangthongkum-2019-341-36	36	broken, split, separate		
+Luangthongkum-2019-341-37	37	buffalo	934	BUFFALO
+Luangthongkum-2019-341-38	38	buy	1869	BUY
+Luangthongkum-2019-341-39	39	caress		
+Luangthongkum-2019-341-40	40	carry (on the back)	465	CARRY ON BACK
+Luangthongkum-2019-341-41	41	caterpillar	2247	CATERPILLAR
+Luangthongkum-2019-341-42	42	catfish	3777	CATFISH
+Luangthongkum-2019-341-43	43	carapace, shell	598	SHELL
+Luangthongkum-2019-341-44	44	centipede	1974	CENTIPEDE
+Luangthongkum-2019-341-45	45	chameleon	1555	CHAMELEON
+Luangthongkum-2019-341-46	46	charcoal	1188	CHARCOAL
+Luangthongkum-2019-341-47	47	chicken	1318	CHICKEN
+Luangthongkum-2019-341-48	48	chili pepper, pungent	919	CHILI PEPPER
+Luangthongkum-2019-341-49	49	chin, jaw	1510	CHIN
+Luangthongkum-2019-341-50	50	cicada	2353	CICADA
+Luangthongkum-2019-341-51	51	civet cat	3157	CIVET
+Luangthongkum-2019-341-52	52	classifier (hand of bananas)		
+Luangthongkum-2019-341-53	53	classifier for flat objects		
+Luangthongkum-2019-341-54	54	classifier for people, persons		
+Luangthongkum-2019-341-55	55	classifier for a piece of meat, cake, etc.		
+Luangthongkum-2019-341-56	56	classifier for plants, e.g. bamboo, rice, banana etc. (clumps, clusters)		
+Luangthongkum-2019-341-57	57	classfier for round objects		
+Luangthongkum-2019-341-58	58	classifier for long objects		
+Luangthongkum-2019-341-59	59	classifier for four-legged animals		
+Luangthongkum-2019-341-60	60	close (one’s eyes)	2063	CLOSE (EYES)
+Luangthongkum-2019-341-61	61	comb (v.)	2070	COMB (VERB)
+Luangthongkum-2019-341-62	62	cotton thread	1161	THREAD
+Luangthongkum-2019-341-63	63	cough (v.)	879	COUGH
+Luangthongkum-2019-341-64	64	crab	2348	CRAB
+Luangthongkum-2019-341-65	65	cricket	2354	CRICKET
+Luangthongkum-2019-341-66	66	crocodile	1857	CROCODILE
+Luangthongkum-2019-341-67	67	crop (of fowl), gizzard		
+Luangthongkum-2019-341-68	68	cross (v.)	574	TRANSIT
+Luangthongkum-2019-341-69	69	crossbow	3193	CROSSBOW
+Luangthongkum-2019-341-70	70	crow (v.)	2074	CROW (VERB)
+Luangthongkum-2019-341-71	71	crush (v.)	1520	CRUSH
+Luangthongkum-2019-341-72	72	cucumber	496	CUCUMBER
+Luangthongkum-2019-341-73	73	dark, late evening		
+Luangthongkum-2019-341-74	74	daughter-in-law	1057	DAUGHTER-IN-LAW
+Luangthongkum-2019-341-75	75	day	1225	DAY (NOT NIGHT)
+Luangthongkum-2019-341-76	76	dead,die	2101	BE DEAD OR DIE
```

Comment:
Space after comma

Reply:
This case and the one above reflect the way the data is represented in the source so I'd leave it as it is.